blog/_posts/2016-07-15-team-joda-says-hello.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Team joda says hello
+layout: post
+created_at: Thu Jul 14 2016
+permalink: blog/2016-07-15-team-joda-says-hello
+current: blog
+author: Johanna Lang and Dayana Mick
+twitter: Teamjoda2016
+categories:
+- student-posts
+---
+
+
+
+# “Always two there are, no more, no less.”
```

Comment:
not sure if this header is meant to be a `h1` (#) or a `h2` like the other quotes (##). 
If this was on purpose, no need to change it of course :)